example/src/release_tests/swipeableReanimation/index.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { Text, Animated as Laminated, StyleSheet } from 'react-native';
```

Comment:
Is there a story behind `Laminated` 😄?

Anyway, If you need to use both Animated and Reanimated in the same file, I'd stick to keeping Animated as Animated, and import Reanimated as Reanimated.

Reply:
I'm sorry, i never thought i'd upload that example and forgot to change the names 🫢